strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexManager.java

```diff
@@ -137,18 +137,29 @@ public void setIndexes(Map<String, RepositoryIndexer> indexes)
 
     public RepositoryIndexer getRepositoryIndexer(String contextId)
     {
-        return indexes.get(contextId);
+        RepositoryIndexer repositoryIndexer = indexes.get(contextId);
+        logger.debug(
+                "RepositoryIndexManager = [{}]. Getting repository indexer. ContextId = [{}], RepositoryIndexer = [{}]",
+                this, contextId, repositoryIndexer);
+        return repositoryIndexer;
     }
 
-    public RepositoryIndexer addRepositoryIndexer(String repositoryId,
+    public RepositoryIndexer addRepositoryIndexer(String contextId,
                                                   RepositoryIndexer value)
     {
-        return indexes.put(repositoryId, value);
+        logger.debug(
```

Comment:
Please. always have the first parameter on the same line and indent the chopped parameters against it.
(Could you please fix this across the whole pull?)

Also, logging operation blocks should be separate by a line before and after... For example:

```
doSomething();
i++;
logger.debug(...);
logger.debug(...);
doSomethingElse();
```

Should look like:

```
doSomething();
i++;

logger.debug(...);
logger.debug(...);

doSomethingElse();
```